workers/unity/Assets/Playground/Scripts/MonoBehaviours/SpawnCubeCommandReceiver.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections.Generic;
+using Generated.Improbable;
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.Commands;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker.Core;
+using UnityEngine;
+using Transform = Generated.Improbable.Transform.Transform;
+
+#region Diagnostic control
+
+// Disable the "variable is never assigned" for injected fields.
+#pragma warning disable 649
+
+// ReSharper disable PossibleInvalidOperationException
+
+#endregion
+
+namespace Playground.MonoBehaviours
+{
+    public class SpawnCubeCommandReceiver : MonoBehaviour
+    {
+        [Require] private Transform.Requirables.Reader transformReader;
+        [Require] private CubeSpawner.Requirables.CommandRequestHandler commandRequestHandler;
```

Comment:
better to name the variable according to which command

Reply:
I could go for cubeSpawnerCommandRequestHandler